docs/generate_format_table.py

```diff
@@ -0,0 +1,159 @@
+import large_image
+from pathlib import Path
+from format_examples_datastore import EXAMPLES_FOLDER, format_examples, fetch_all
+
+
+TABLE_FILE = Path('./format_table.rst')
+NO_MULTIFRAME_SOURCES = ['deepzoom', 'openjpeg', 'openslide']
+
+
+def evaluate_examples():
+    large_image.tilesource.loadTileSources()
+    available_tilesources = large_image.tilesource.AvailableTileSources
+    results = []
+    for format_data in format_examples:
+        name = format_data.get('name')
+        reference = format_data.get('reference')
+        for example in format_data.get('examples', []):
+            skip = example.get('skip')
+            if not skip:
+                filename = example.get('filename')
+                url = example.get('url')
+                extension = filename.split('.')[-1]
+                filepath = Path(EXAMPLES_FOLDER, filename)
+                print(f'Evaluating {filename}. ')
+                for tilesource_name, readable in large_image.canReadList(filepath):
+                    tilesource = available_tilesources.get(tilesource_name)
+                    if readable and tilesource:
+                        try:
+                            s = tilesource(filepath)
+                            results.append(
+                                dict(
+                                    name=name,
+                                    reference=reference,
+                                    extension=extension,
+                                    filename=filename,
+                                    url=url,
+                                    tilesource=tilesource_name,
+                                    multiframe=(
+                                        False if tilesource_name in NO_MULTIFRAME_SOURCES else
+                                        True if s.getMetadata().get('frames') is not None else
+                                        'Maybe; no multiframe sample found.'
```

Comment:
I'm wondering if we should just leave the maybes blank instead of saying maybe.

Reply:
Changed to "Unknown" per @jeffbaumes' suggestion in [2f65780](https://github.com/girder/large_image/pull/1622/commits/2f65780fb164e0198781de5adfafa03dd033b0d6)